package.json

```diff
@@ -6,11 +6,15 @@
   "main": "index.js",
   "files": [
     "contracts",
-    "test"
+    "test",
+    "build"
   ],
   "scripts": {
+    "preversion": "npm run restore",
+    "restore": "rm -rf build && npm run compile && tnt iN && tnt cB",
     "test-norpc": "truffle test",
     "test": "run-with-testrpc -l 20000000 --noVMErrorsOnRPCResponse true 'truffle test'",
+    "install": "cd $INIT_CWD && npm explore truffle -- npm install solc@0.5.0",
```

Comment:
maybe use 0.5.3, since that is the version we use in the safe now

Reply:
okk, I'm gonna update it